### DISCUSSION TITLE
why this `onwheel` event not triggered?

### DISCUSSION THREAD

**Discussion by daGaiGuanYu** [solidjs/solid#2213] [Q&A] [answered]
When I bind event like this, the click event is triggered normally, but the wheel event wouldn't be triggered.

``` js
import { onMount } from 'solid-js'

function App() {
  let div = <div style="width: 100px; height: 100px; background: black" />
  div.onwheel = console.warn // not triggered
  div.onclick = console.log // triggered
  return div
}
```

And this way, both triggered
``` js
function App() {
  let div
  onMount(() => {
    div.onwheel = console.warn
    div.onclick = console.log
  })
  return <div ref={div} style="width: 100px; height: 100px; background: black" />
}
```

Can anybody help me about this? THX!!!  
It's been breaking me all day.

**Comment by mdynnl** [comment, marked as answer]
it's a [chrome bug](https://issues.chromium.org/issues/345948299) initially reported on [discord](https://discord.com/channels/722131463138705510/1248651749229858896)

**Reply by daGaiGuanYu** [reply]
OOOOk, THX! really
